app/services/food_service.rb

```diff
@@ -0,0 +1,18 @@
+class FoodService 
+
+  class << self
+
+    def conn 
+      Faraday.new(url: "https://api.nal.usda.gov/fdc/v1/") do |faraday|
+        faraday.headers["X-API-KEY"] = ENV['food_api_key']
+      end
+    end
+
+    def foods_by_search(food)
```

Comment:
I wonder if you could add additional params here so that you could get a page size and set it to 10